### DISCUSSION TITLE
Editor: Flipping across playfield axis

### DISCUSSION THREAD

**Discussion by Okorin** [ppy/osu#15064] [Ideas] [answered]
Yo, so as I kind-of implied on my twitter, I tried out making a map with the lazer editor and wanted to somehow dump my experiences somewhere so here goes. All this was done on **osu! 2021.1006.1-lazer**

I noticed (or couldn't find) that there's any way to rotate an object across the playfield or the axis. By default it flips your current selection within the box that is selected, which makes using symmetry in any mapping very tricky because right now i'm essentially just eyeballing it

(so e.g. making something like this required me to just eyeball whatever this is after )
![example](https://i.imgur.com/mLFYmlv.png)

would also be cool to have that somehow and i couldn't find an open issue for it so if i'm doing this wrong please lmk

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15071
